src/pages/edit/index.js

```diff
@@ -0,0 +1,236 @@
+import { Route } from "react-router-dom";
+import Card from "../../components/card";
+import TextInput from "../../components/form/textInput";
+import "./style.css";
+import "../../styles/index.css";
+import LockIcon from "../../assets/icons/locIcon";
+
+const hrStyle = {
+  backgroundColor: "e6ebf5",
+  height: 1,
+};
+
+const EditProfile = () => {
+  return (
+    <>
+      <main>
+        <h2>Profile</h2>
+        <Card>
+          <div className="profile-container">
+            <div className="profile-icon">
+              <p>AJ</p>
+            </div>
+            <div className="profile-summary">
+              <h3>Full name</h3>
+              <p>Title</p>
+            </div>
+          </div>
+
+          <div className="main-info-grid">
+            <div className="basic-info">
+              <hr style={hrStyle} />
```

Comment:
Be sure to update this when you make the css class

Reply:
Corrected